src/parsimmon.js

```diff
@@ -30,8 +30,10 @@
     this._ = action;
   };
 
-  Parsimmon.Parser = Parser;
-
+  Parsimmon.isParser = function (obj) {
```

Comment:
If you change this to `var isParser = Parsimmon.isParser = function(obj) { ...` then we can use it inside the `assertParser` function, which I think would be nice, if you don't mind changing that too.